component/alert/alert.css

```diff
@@ -0,0 +1,64 @@
+.alert-container {
+  display: flex;
+  flex-direction: column;
+  margin-left: 2rem;
+  padding: 1.5rem;
+  padding-bottom: 2.5rem;
+  font-family: var(--primary-font);
+}
+.alert-heading {
+  margin-top: 1.5rem;
+  padding-bottom: 1rem;
+  font-size: 3rem;
+  color: black;
+}
+
+.alert-description {
+  margin-bottom: 1.5rem;
+  font-size: 1.5rem;
+  color: var(--secondary-text);
+}
+
+.alert-title {
+  font-size: 2rem;
+  padding-top: 1.5rem;
+  color: black;
+}
+.alert {
+  display: flex;
```

Comment:
if using flex property repeatedly, maybe make that a class.

Reply:
done